settings.gradle

```diff
@@ -1,3 +1,4 @@
+include ':picasso-provider'
```

Comment:
Should be between `pollexor` and `sample`

Reply:
Updated.